pytorch_lightning/loggers/base.py

```diff
@@ -452,7 +453,8 @@ def name(self):
     def version(self):
```

Comment:
the return values for name/version don't match the base interface

Reply:
don't know how to cleanly address this. 
one could make version and name optional in the base class, but then again they are not really optional

Reply:
For the purposes of the dummy logger, should we return empty string and 0?